src/features/pages/home/cta/index.tsx

```diff
@@ -37,9 +38,9 @@ const CTA = () => {
                 <Heading.H2 className="text-solid-slate-50">
                     Join over 2.5 million online traders
                 </Heading.H2>
-                <Button size="lg" className="hidden lg:block">
+                <LinkButton size="lg" className="hidden lg:block" href="/signup">
```

Comment:
this should be `CustomLink` @fasihali-deriv and @jia-deriv

Reply:
pull the latest from `translation-new-homepage` using `TradersHubCtaButton` now